modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormFieldTypesServlet.java

```diff
@@ -115,8 +124,27 @@ protected void doGet(
 			httpServletResponse, fieldTypesJSONArray.toJSONString());
 	}
 
+	protected Object getFieldProperties(
+		String ddmFormFieldName, HttpServletRequest httpServletRequest) {
+
+		if (StringUtil.equals(ddmFormFieldName, "rich_text")) {
+			EditorConfiguration richTextEditorConfiguration =
+				EditorConfigurationFactoryUtil.getEditorConfiguration(
+					StringPool.BLANK, ddmFormFieldName, "ckeditor",
+					Collections.emptyMap(),
+					(ThemeDisplay)httpServletRequest.getAttribute(
+						WebKeys.THEME_DISPLAY),
+					RequestBackedPortletURLFactoryUtil.create(
+						httpServletRequest));
+
+			return richTextEditorConfiguration.getData();
+		}
+
+		return JSONFactoryUtil.createJSONObject();
```

Comment:
With this most fields will have an empty JSON object. Can ew put the check before we call this method? That way we save some data transfer.